src/booleans.js

```diff
@@ -1,61 +1,66 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a&&b
 };
 
 function either(a, b) {
-  // your code here
+  return a||b
 };
 
 function none(a, b) {
-  // your code here
+  return !a&&!b
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b || !a&b)? true:false;
```

Comment:
This could be simplified as above by just returning the result of the expression:
```
  return (a && b === false) || (b && a === false);
```